packaging/datadog-agent/source/setup_agent.sh

```diff
@@ -389,6 +389,26 @@ tar -xz -C "$DD_HOME/agent" --strip-components 1 -f "$DD_HOME/agent.tar.gz"
 rm -f "$DD_HOME/agent.tar.gz"
 print_done
 
+print_console "* Downloading integrations from GitHub (~5 MB)"
+mkdir -p "$DD_HOME/3rd-party"
+$DOWNLOADER "$DD_HOME/integrations.tar.gz" "https://api.github.com/repos/DataDog/integrations-core/tarball"
```

Comment:
when we end up versioning the integrations core with the version number of the agent, we can download it like this:

`https://api.github.com/repos/DataDog/integrations-core/tarball/{version}`

see more here: https://developer.github.com/v3/repos/contents/#get-archive-link

`GET /repos/:owner/:repo/:archive_format/:ref`